go.mod

```diff
@@ -1,9 +1,6 @@
 module github.com/containers/common
 
-go 1.21
-
-// Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates
-toolchain go1.21.0
```

Comment:
This should stay so we have the comment to remind us to never bump this further than the minimum go version.

Reply:
Then we need to fix the tools built into the Makefile to not remove the line or re-add it when it gets removed.  Very easy to remove this line.